jdisc_core/src/main/java/com/yahoo/jdisc/Request.java

```diff
@@ -48,6 +48,11 @@
     private boolean serverRequest;
     private Long timeout;
     private URI uri;
+    private RequestType requestType;
+
+    public enum RequestType {
```

Comment:
`RequestType` is a too generic term in my opinion. Consider naming the concept something more specific.

Reply:
That's what it's already called in the metric so using a different name here increases entropy a bit. Did you have a suggestion in mind?

Reply:
Not really, good naming ain't easy :)